playbooks/SOC Simulator/Create Zero day Malicious File Indicator.json

```diff
@@ -0,0 +1,126 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Create Zero day Malicious File Indicator",
+    "aliasName": null,
+    "tag": null,
```

Comment:
Why we need this PB in SP? This should be part of some scenario SP, right?

Reply:
Hi @cs-suyog-jinturkar 

I have removed this playbook. Also made below changes

- Removed extra playbooks from the playbook collection
   > Create Alert
   > Create Phishing Alert
   > Create Zero day Malicious File Indicator
   > Get Malicious IP and URL
   > Replace Variables Values

- Updated Playbook collection name to "02 - Use Case - SOC Simulator"

- Updated info.json accordingly